inc/run_cppunittest_suite.h

```diff
@@ -0,0 +1,24 @@
+#ifndef RUN_CPPUNITTEST_SUITE_H
+#define RUN_CPPUNITTEST_SUITE_H
+
+#include "macro_utils/macro_utils.h"
+#include "ctest.h"
+#include "cppunittest.h"
+
+using namespace Microsoft::VisualStudio::CppUnitTestFramework;
+
+#define RUN_TESTS_WITH_CPP_UNITTEST(test_suite) \
+    extern C_LINKAGE const TEST_FUNCTION_DATA MU_C2(TestListHead_, test_suite); \
+    static const TEST_FUNCTION_DATA* entry_test_function_data = &MU_C2(TestListHead_, test_suite); \
```

Comment:
are these used?

Reply:
No, removing.

---
In reply to: [608146360](https://github.com/Azure/c-testrunnerswitcher/pull/65#discussion_r608146360) [](ancestors = 608146360)

Reply:
Actually they are needed to force linking

---
In reply to: [608153573](https://github.com/Azure/c-testrunnerswitcher/pull/65#discussion_r608153573) [](ancestors = 608153573,608146360)